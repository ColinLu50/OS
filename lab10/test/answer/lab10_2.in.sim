4480
665
687
650
665
637
