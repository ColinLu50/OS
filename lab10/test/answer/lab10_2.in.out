FCFS
11 -> 533 -> 343 -> 469 -> 613 -> 622 -> 245 -> 260 -> 676 -> 585 -> 135 -> 350 -> 90 -> 294 -> 395 -> 150 -> 362 -> 39 -> 395 -> 619
4480
SSTF
11 -> 39 -> 90 -> 135 -> 150 -> 245 -> 260 -> 294 -> 343 -> 350 -> 362 -> 395 -> 395 -> 469 -> 533 -> 585 -> 613 -> 619 -> 622 -> 676
665
SCAN
11 -> 19 -> 676 -> 622 -> 619 -> 613 -> 585 -> 533 -> 469 -> 395 -> 362 -> 350 -> 343 -> 294 -> 260 -> 245 -> 150 -> 135 -> 90 -> 39 -> 11 -> 0 -> 39 -> 90 -> 135 -> 150 -> 245 -> 260 -> 294 -> 343 -> 350 -> 362 -> 395 -> 469 -> 533 -> 585 -> 613 -> 619 -> 622 -> 676
687
C-SCAN
11 -> 19 -> 676 -> 622 -> 619 -> 613 -> 585 -> 533 -> 469 -> 395 -> 362 -> 350 -> 343 -> 294 -> 260 -> 245 -> 150 -> 135 -> 90 -> 39 -> 11 -> 0 -> 678 -> 676 -> 622 -> 619 -> 613 -> 585 -> 533 -> 469 -> 395 -> 362 -> 350 -> 343 -> 294 -> 260 -> 245 -> 150 -> 135 -> 90 -> 39
650
LOOK
11 -> 19 -> 676 -> 622 -> 619 -> 613 -> 585 -> 533 -> 469 -> 395 -> 362 -> 350 -> 343 -> 294 -> 260 -> 245 -> 150 -> 135 -> 90 -> 39 -> 11 -> 39 -> 90 -> 135 -> 150 -> 245 -> 260 -> 294 -> 343 -> 350 -> 362 -> 395 -> 469 -> 533 -> 585 -> 613 -> 619 -> 622 -> 676
665
C-LOOK
11 -> 19 -> 676 -> 622 -> 619 -> 613 -> 585 -> 533 -> 469 -> 395 -> 362 -> 350 -> 343 -> 294 -> 260 -> 245 -> 150 -> 135 -> 90 -> 39 -> 11 -> 676 -> 622 -> 619 -> 613 -> 585 -> 533 -> 469 -> 395 -> 362 -> 350 -> 343 -> 294 -> 260 -> 245 -> 150 -> 135 -> 90 -> 39
637
